apps/oc_chef_authz/src/oc_chef_policy.erl

```diff
@@ -0,0 +1,180 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Stephen Delano <stephen@opscode.com>
```

Comment:
Did Stephen really write this? ;) Also, we should use the apache header, the Chef Software name, and the right year.